packages/mobile/lib/routers/helper.ts

```diff
@@ -70,6 +72,7 @@ function initialiseOnboardingSubrouters(): void {
 }
 
 function initialiseWalletSubrouters(): void {
```

Comment:
Same here, I would rename this even though is not part of the scope of this PR, I guess we forgot about it in the refactor 🙏🏼